data/templates/nginx/plain/yunohost_admin.conf

```diff
@@ -21,9 +21,13 @@ server {
     ssl_session_cache shared:SSL:50m;
     ssl_prefer_server_ciphers on;
     ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
-    ssl_ciphers ALL:!aNULL:!eNULL:!LOW:!EXP:!RC4:!3DES:+HIGH:+MEDIUM;
+    ssl_ciphers "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH";
+    ssl_ecdh_curve secp384r1;
+    ssl_stapling on;
+    ssl_stapling_verify on;
 
     add_header Strict-Transport-Security "max-age=31536000;";
+    add_header X-Content-Type-Options nosniff;
```

Comment:
didn't know about this one, but seems relevant : http://stackoverflow.com/questions/18337630/what-is-x-content-type-options-nosniff